spark/src/main/java/org/apache/iceberg/spark/data/SparkOrcReader.java

```diff
@@ -104,6 +104,7 @@ private ReadBuilder(Map<Integer, ?> idToConstant) {
           return OrcValueReaders.floats();
         case DOUBLE:
           return OrcValueReaders.doubles();
+        case TIMESTAMP:
         case TIMESTAMP_INSTANT:
           return SparkOrcValueReaders.timestampTzs();
```

Comment:
Does SparkOrcValueReaders.timestampTzs() read both LocalDateTime and OffsetDateTime?

Reply:
Spark uses longs (microseconds since epoch) to store timestamps, so SparkOrcValueReaders.timestampTzs() just needs to return longs.